src/Sylius/Behat/Client/ApiClientInterface.php

```diff
@@ -40,6 +40,8 @@ public function applyTransition(string $id, string $transition, array $content =
 
     public function customItemAction(string $id, string $type, string $action): Response;
 
+    public function customAction(string $type, string $action): Response;
```

Comment:
```suggestion
    public function customAction(string $method, string $url): Response;
```
and these arguments could be in the reverse order imo

Reply:
However, URL could suggest that we are providing a full path here. Not sure, if action is not a better wording.

Reply:
And in the implementation this argument is used as a full path